packages/next/build/webpack/loaders/next-middleware-asset-loader.ts

```diff
@@ -0,0 +1,20 @@
+import { getModuleBuildInfo } from './get-module-build-info'
+import crypto from 'crypto'
+
+export default function MiddlewareAssetLoader(this: any, source: Buffer) {
+  const name = sha1(source)
```

Comment:
Why are we using `sha1` here, can we use the built-in webpack hash handling similar to the `next-image-loader`? 

x-ref: https://github.com/vercel/next.js/blob/68047112d6ad9ac293161ecd3d594b05eda4d258/packages/next/build/webpack/loaders/next-image-loader.js#L14